packages/fakta-ny-inntekt/src/components/tilkommetAktivitet/TilkommetAktivitetPanel.tsx

```diff
@@ -55,7 +55,7 @@ export const TilkommetAktivitetPanel = ({
     control,
     name: `VURDER_TILKOMMET_AKTIVITET_FORM.${formFieldIndex}.perioder`,
   });
-  fields.sort((a, b) => dayjs(a.fom).diff(dayjs(b.fom)));
+  fields.sort(sortPeriodsByFom);
```

Comment:
`fields.sort` muterer arrayet, kan det skape problemer med React Hook Form sin state?
```suggestion
  const sortedFields = [...fields].sort(sortPeriodsByFom);
```

og 

```
    fields={sortedFields}
```
lenger ned